app/src/main/res/layout/screen_patients.xml

```diff
@@ -63,21 +63,6 @@
         android:src="@drawable/illustrations_homescreen"
         tools:ignore="ContentDescription" />
 
-      <FrameLayout
```

Comment:
As discussed, once you will use a duplicate xml for `screen_patients` for your development, this commit won't be necessary.